### PR TITLE
Honda Civic 2022: fix spotty camera query

### LINKED ISSUE #31946: Honda Radarless: fix spotty camera query & understand extra ECU
Some context here: https://github.com/commaai/openpilot/issues/28483

Basically, most/all Civic 2022 (radarless) have the ACC (marked as `fwdRadar` since the address is the same as the radar on other cars) ECU sometimes totally ignore our FW query. This is one of the only ECUs available from the camera without the OBD-II port, so we should fix this.

There's also another ECU at 0x18DAB3F1 with a part number of `8S108` (`8S102` on HR-V radarless), and on the HR-V radar is mostly missing with this other mystery ECU only being present. I searched online and only found TSB's about the `36161` part we have for the 2022 Civic, not `8S108`. Certain part sites claim `8S103` is a "Camera Set Mono," and `36160` is "Camera Monocular - Honda," but I'm not sure why we're only now seeing this 8S part in the FW

With some testing with @MarcoTheDingo, it may be possible this ECU doesn't respond immediately after turning car on. Do we need CAN fingerprinting for these platforms?

### PR DESCRIPTION
**Pull request by sshane**

### CODE REVIEW

File: launch_openpilot.sh [hunk 1]
```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/bash
 
+DISABlE_FW_CACHE=1
```
Comment (adeebshihadeh) [inline launch_openpilot.sh:3]: should we just remove the cache? do unplugged DSU people still need it?

Comment (sshane) [reply launch_openpilot.sh:3]: I believe so, yes

Comment (jyoung8607) [inline launch_openpilot.sh:3]:
```suggestion
DISABLE_FW_CACHE=1
```

File: launch_openpilot.sh [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+
```
Comment (sshane) [inline launch_openpilot.sh:4]:
```suggestion
```

**Pull request merged**